forge/forge/compile.py

```diff
@@ -5,6 +5,7 @@
 from typing import Optional, List, Dict, Any, Tuple, Union
 from dataclasses import dataclass, field
 
+from forge.optimizers import Optimizer
```

Comment:
Let's move this import a bit lower. 

It'll be good to strive to have unified import semantics:
- Built-in python libs
- Third-party libs
- Our libs